backend/app/models/__init__.py

```
@@ -39,6 +39,9 @@
     Segment,
 )
 
+# Import segment models
+from .segments import ContentSegment, ContentSegmentBase
```

Ruff (0.12.2)

`.segments.ContentSegment` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)